tfhe/src/integer/server_key/radix_parallel/tests_cases_signed.rs

```diff
@@ -1931,3 +1933,600 @@ where
         }
     }
 }
+
+pub(crate) fn signed_default_left_shift_test<P, T>(param: P, mut executor: T)
```

Comment:
I've totally removed tests_cases_signed on my `scalar_mul` branch, these functions should go to `tests_signed/test_shift.rs` and `tests_signed/test_rotate.rs` instead.

Reply:
(to avoid conflicts)

Reply:
Ok! I think it is right now, but I had to duplicate code. Is that what you had in mind?